RELEASE.md

```diff
@@ -23,3 +23,24 @@ make chlog-update
 ```
 
 This will delete all entries (other than the template) in the `.chloggen` directory and create a populated Changelog.md entry. Make sure that the PR you are generating for the release has the `[chore]` prefix, otherwise CI will not pass.
+
+
+## Release managers
+
+A release manager is the person responsible for a specific release. While the manager might request help from other folks, they are ultimately responsible for the success of a release.
+
+In order to have more people comfortable with the release process, and in order to decrease the burden on a small number of volunteers, all approvers and maintainers are release managers from time to time, listed under the Release Schedule section. That table is updated at every release, with the current manager adding themselves to the bottom of the table, removing themselves from the top of the table.
+
+## Release schedule
+
+The operator should be released within a week after the [OpenTelementry collector releas](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/release.md#release-schedule).
+
+| Version | Release manager |
+|---------------------------|
+| v0.69.0 | @pavolloffay    |
+| v0.70.0 | @jpkrohling     |
```

Comment:
I guess it's my time now to step out from maintainership. I was already away for a while, just lurking around in case something needed someone to break a deadlock, but there are enough maintainers now so this problem won't happen. I also trust that the current maintainers are doing a great job, and I don't deserve to be called a "maintainer" and be compared with you, the ones actually doing the work.

Reply:
Thanks for bootstrapping the project and all the work. 

I can wait for your PR if you prefer.

Reply:
Thank you @jpkrohling ❤️ @pavolloffay If you'd like, you can change his name for mine.

Reply:
> Thank you @jpkrohling heart @pavolloffay If you'd like, you can change his name for mine.

Ups, let me revert what i did :roll_eyes: :x: